### ISSUE TITLE
Drawing Box Issue

### ISSUE THREAD

**Issue by Haoo906** [sjtuplayer/Compositional_Neural_Painter#8]
it possible to change the shape of the drawing boundary (from a fixed rectangle) to draw an area of arbitrary shape?

**Comment by sjtuplayer** [comment]
Of course. The simplest way is to use a mask to represent the area you want to draw. In the new version, the compositor can predict a mask with arbitrary shape instead of the parameter of the drawing box. And the painter network should also be retrained to predict strokes inside a given mask. 
However, by my experience, if the painter netowrk aims to paint in an irregular shape, the L2 loss would not be as low as the original one which predicts paths in a squared box. But you can have a try and wish you a good luck!
